components/package.json

```diff
@@ -43,9 +43,8 @@
   },
   "peerDependencies": {
     "@material-ui/core": "^4.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "@pxblue/colors": "^2.0.0"
+    "@pxblue/colors": "^2.0.0",
+    "react": "^16.8.6"
```

Comment:
do we not have a peer dependency on react-dom anymore?

Reply:
Great catch! This was an accidental edit.